src/app/modals/new-ig-modal/new-ig-modal.component.ts

```diff
@@ -147,10 +141,32 @@ export class NewIgModal implements OnInit {
     }
   }
 
+  ngAfterViewInit() {
```

Comment:
@codiak This is a great approach and pattern to handling initial modal focus across all modals!